src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -27,6 +27,7 @@ public sealed partial class QuicConnection : System.IDisposable
         public System.Net.Security.SslApplicationProtocol NegotiatedApplicationProtocol => throw null;
         public ValueTask CloseAsync(long errorCode, System.Threading.CancellationToken cancellationToken = default) => throw null;
         public void Dispose() => throw null;
+        public static bool IsQuicSupported => throw null;
```

Comment:
We're still operating in a "none of this has been reviewed yet and we won't ship these APIs until we do and revise them accordingly" mode, right? Is there a must-do issue tracking that to make sure it doesn't slip through the cracks?

Reply:
> We're still operating in a "none of this has been reviewed yet and we won't ship these APIs until we do and revise them accordingly" mode, right?

Yes.

> Is there a must-do issue tracking that to make sure it doesn't slip through the cracks?

It's been in active discussion during QUIC syncs; I'm aiming for us to have that ironed out in the next couple weeks. https://github.com/dotnet/runtime/issues/31709